pgcli/pgcompleter.py

```diff
@@ -45,12 +46,31 @@ def __init__(self, smart_completion=True):
         super(self.__class__, self).__init__()
         self.smart_completion = smart_completion
 
+        self.name_pattern = compile("^[_a-z][_a-z0-9\$]*$")
+
+    def extend_escape_name(self, name):
+        if not self.name_pattern.match(name) or name in self.keywords or name in self.functions:
```

Comment:
`upper()`/`lower()` -> `potato`/`tomato`. It's all the same. 

But seriously you're right it should be upper(). I was just a sleepy kitten when I left that comment. But I see you've sent a new PR with the fixes. :+1: